iaso/api/org_unit_change_requests/serializers.py

```diff
@@ -177,7 +178,7 @@ class OrgUnitChangeRequestWriteSerializer(serializers.ModelSerializer):
     """
 
     uuid = serializers.UUIDField(required=False, default=uuid.uuid4)
-    org_unit_id = serializers.PrimaryKeyRelatedField(
+    org_unit_id = IdOrUuidRelatedField(
         source="org_unit",
         queryset=OrgUnit.objects.all(),
         required=False,
```

Comment:
It is required. Otherwise, to which OrgUnit do you assign this change request?

Reply:
It's used for both `POST` and `PUT` and `org_unit_id` isn't required on update.
But we can make it required, WDYT?